webapp/lib/charts.dart

```diff
@@ -130,25 +133,29 @@ class DailyTimeseriesLineChartView {
             ..distribution = 'linear'
             ..bounds = 'ticks'
             ..time = (new chartjs.TimeScale(unit: 'day'))
-        ],
-        yAxes: [
-          new chartjs.ChartYAxe()
-            ..ticks = (new chartjs.LinearTickOptions()..beginAtZero = true)
+      ],
+      yAxes: [
+        new chartjs.ChartYAxe()
+          ..ticks = (new chartjs.LinearTickOptions()..beginAtZero = true)
         ]),
       hover: new chartjs.ChartHoverOptions()..animationDuration = 0
     );
 
-    var chartConfig = new chartjs.ChartConfiguration(type: 'line', data: chartData, options: chartOptions);
+    var chartConfig = new chartjs.ChartConfiguration(
```

Comment:
accidental/automatic line split? please revert

[we don't use a fixed limit for max line length in Dart files, and from my experience 80 is too low, 120 is about right, but because sometimes you may need a line longer than that, we tend to split the lines manually. I'd recommend turning the auto-formatter off, if you find it helpful for indenting brackets etc, just disable the max line length setting.]

Reply:
OKay. I've reverted. I'll turn off the auto-formatter. It's messing up a lot